solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -343,17 +345,45 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
     }
 
     /**
-     * @dev returns the reserve balances of the given reserve tokens
+     * @dev transfers a portion of the accumulated conversion fees
+     */
+    function processNetworkFees() external protected {
```

Comment:
add more descriptive comment, such as "calculates and transfers a portion of the fees to the network fee wallet"

Reply:
done